packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta/request-access-token/index.md

```diff
@@ -1,48 +1,43 @@
-
-    The response that is returned looks something like this (the token is truncated for brevity):
-
-    ```bash
-    http://{yourOktadomain}/authorization-code/callback#access_token=eyJraWQiOiJEa1lUbmhTdkd5OEJk.......Y5D6XG&token_type=Bearer&expires_in=3600&scope=okta.users.read&state=1234
-    ```
+    * **Callback URL**: Define the callback location where Okta returns the token after the user finishes authenticating. This URL must match one of the redirect URIs that you configured in the <GuideLink link="../create-an-OAuth-2.0-app-in-okta">Create an OAuth 2.0 app in Okta</GuideLink> section.
+    * **Auth URL**: Enter the authorization endpoint for your Org Authorization Server. For example: `https://{yourOktadomain}/oauth2/v1/authorize?none=WBYA`
+        > **Note:** There isn't an entry in the dialog box for a `nonce` parameter. But, you can append it to the Auth URL here. Use any value for `nonce`. This is a string that is included in the returned ID token. It associates a client session with an ID token and mitigates replay attacks. In this example, we aren't requesting the `id_token` response type, so an ID token isn't returned.
+    * **Client ID**: Use the `client_id` of your Okta OAuth application that you created in the <GuideLink link="../create-oauth-app">previous step</GuideLink>.
+    * **Scope**: Include the scopes that allow you to perform the actions on the endpoint that you want to access. The scopes requested for the access token must already be in the application's grants collection and the user must have the permission to perform those actions. See <GuideLink link="../scopes">Scopes & supported endpoints</GuideLink> for more information.
+    * **State**: Use any alphanumeric value. The authorization server reflects this string when redirecting the browser back to the client, which helps prevent cross-site request forgery.
+    * **Client Authentication**: Leave the default of **Send as Basic Auth Header**.
 
+7. Click **Request Token**. You are prompted to sign in to your Okta org. After you are authenticated, the **Manage Access Tokens** window displays the access token, including the scopes requested. The token also automatically populates the **Available Token** drop-down list.
     > **Note:** The lifetime for this token is fixed at one hour.
+8. Click **Use Token** at the bottom of the window to use this access token in your request to the `/users` endpoint.
+9. Click **Send**. Since you requested `okta.users.read`, the response should contain an array of all the users associated with your app. This is dependent on the user's permissions.
 
-4. Copy the access token for use in subsequent <GuideLink link="../save-access-token">requests to Okta API endpoints</GuideLink>.
+> **Note:** You can also manually build the request URL and paste it into a private browser window. After you authenticate, the browser returns the access token in the address bar. Your request URL should look something like this:<br>
+    ```
```

Comment:
Don't we need to declare the language at the start of the code snippet?

Reply:
I wanted it to be a part of the note - when I left the language, it showed up in the Note example of the URL. Since it's just a URL, I don't think the language is necessary in this instance.